tests/integration/test_charm.py

```diff
@@ -60,7 +60,7 @@ async def test_build_and_deploy(ops_test: OpsTest, charm: str, test_charm: str)
             trust=True,
         ),
         ops_test.model.deploy(
-            charm,
+            entity_url=str(charm),
```

Comment:
this is already a string
```suggestion
            entity_url=charm,
```